browser/extensions/ethereum_remote_client_util.cc

```diff
@@ -38,14 +38,7 @@ std::string GetInfuraProjectID() {
 }
 
 std::string GetBraveKey() {
-  std::string brave_key(BUILDFLAG(BRAVE_SERVICES_KEY));
-  std::unique_ptr<base::Environment> env(base::Environment::Create());
-
-  if (env->HasVar("BRAVE_SERVICES_KEY")) {
-    env->GetVar("BRAVE_SERVICES_KEY", &brave_key);
-  }
-
-  return brave_key;
+  return BUILDFLAG(BRAVE_SERVICES_KEY);
 }
```

Comment:
There is some infura related stuff above, do we need it?

Reply:
We need for the old Brave Wallet which will still relies on an Infura project ID.